Web page create data

ardanov96 opened the pull request.

ardanov96 committed:
fix export to excel button in Location

ardanov96 committed:
add stored items and date in returns table

ardanov96 committed:
add allowed sorting to stored items

ardanov96 committed:
add code and url

ardanov96 committed:
fix edit webpage & sorting in StoreItemsReturn

ardanov96 committed:
refactor url field

inikoo commented on app/Actions/Traits/Authorisations/Inventory/WithWarehouseEditAuthorisation.php:
roll back this, can not change auth files, explain me in chat why you need this change and i will have a look

inikoo commented on app/Actions/Web/Webpage/UI/CreateWebpage.php:
remove commented code

inikoo reviewed (changes requested):
1. Next time make on PR per issue , done not mix 
2. run pint before commit.  `vendor/bin/pint --dirty`

inikoo committed:
refactor: streamline `WithWarehouseEditAuthorisation` by removing unused `warehouseId` logic and leveraging `$this->warehouse` for permissions

inikoo committed:
Merge branch 'main' into WebPage-CreateData

# Conflicts:
#	app/Actions/Fulfilment/PalletReturn/UI/IndexPalletReturns.php

inikoo committed:
refactor: clean up unused code and improve formatting in pallet returns resources and actions

- Removed commented-out unused code in `TablePalletReturns.vue`
- Enhanced `PalletReturnsResource` with cleaner variable handling for pallets and stored items
- Improved lambda spacing and code readability in `IndexPalletReturns`

inikoo committed:
refactor: adjust import formatting and align variable assignment in `LocationsExport`

- Cleaned up and aligned imports for better readability
- Improved code alignment for `$this->columns` initialization

inikoo committed:
refactor: remove unused commented-out parameter in `EditWebpage` action
